TEP008.rst

```diff
@@ -0,0 +1,46 @@
+TEP008: Tighter integration of Atomic Data, Plasma and Model
+============================================================
+
+Status
+======
+**Discussion**
+
+Responsible
+===========
+
+@wkerzendorf
+@yeganer
+
+Branches and Pull requests
+==========================
+
+Description
+===========
+
+TARDIS current plasma is structured in a very modular way that allows easy
+testing and quick addition of new plasma modules. We restructured the Plasma
+to this new form in 2015 and implemented the most important parts of the
+framework. Due to time constraints, not all functionality was moved to this new
+structure and there is still considerable amount of legacy code. In addition,
+while the functionality is there we lack documentation that explains the new
+rationale to the collaboration and users.
+
+With this TEP we have multiple aims - ensure that all remaining parts that will
+fit within the new structure are moved. Ensure that we update the documentation
+to better reflect the current state of the code. This will include several
+examples on how to write new plasma modules.
+
+Implementation
```

Comment:
I think the code is the more important part here, but yes documentation is a small aspect of it. So we could reorder this.